cpp/src/groupby/hash/groupby.cu

```diff
@@ -27,160 +27,23 @@
 #include <utilities/column_utils.hpp>
 #include <utilities/cuda_utils.hpp>
 #include <cudf/utilities/legacy/type_dispatcher.hpp>
-#include "aggregation_requests.hpp"
+#include "../common/aggregation_requests.hpp"
 #include "groupby.hpp"
 #include "groupby_kernels.cuh"
-#include "type_info.hpp"
-
+#include "../common/type_info.hpp"
+#include "../common/util.hpp"
 #include <rmm/thrust_rmm_allocator.h>
 #include <thrust/fill.h>
 #include <type_traits>
 #include <vector>
 
+using namespace cudf::groupby::common;
```

Comment:
Don't use global "using" statements for namespaces.

Reply:
All right